src/perl/t/data/config/config

```diff
@@ -56,8 +56,10 @@ api_key = 777
 ss_client_id= 888
 
 [irods]
-manifest_path=/Sanger1-dev/home/glsai
-lab_sample_qc_path=/Sanger1-dev/home/glsai
+14MG_manifest_path=/Sanger1-dev/home/glsai
+14MG_lab_sample_qc_path=/Sanger1-dev/home/glsai
+Standard_manifest_path=/Sanger1-dev/home/glsai
+Standard_lab_sample_qc_path=/Sanger1-dev/home/glsai
```

Comment:
It is OK for now, but later if we would have more customers, then I'd rather have a separate JSON config file. It would be much cleaner and readable from my point of view. But don't refactor it now.